src/main/java/com/sampleweb/pages/ComputersAndAccessoriesPage.java

```diff
@@ -0,0 +1,70 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class ComputersAndAccessoriesPage extends HomePage {
+    public String title = "Computers & Accessories";
+    private String pageTitle = "//*[@id='fst-hybrid-dynamic-h1']/div/h1/b";
+    private String globalStore = "//*[@id='leftNav']/ul[3]/div/li/span/span/div/label/span/span/img";
+    private String averageReviewsText = "//*[@id='leftNav']/h4[9]";
+    private String menuText = "//*[@id='nav-subnav']/a[1]";
+    private String dropDownMenu = "//*[@id='searchDropdownBox']";
+    public static String PATH = "https://www.amazon.co.uk/s?i=computers&ref=nb_sb_noss";
+    private String products = "//*[@class='s-result-list s-search-results sg-row']//*[@class='s-include-content-margin s-border-bottom']";
+    private String amazonPrimeCheckbox = "//*[@id='leftNav']/ul[5]/div/li[1]/span/span/div/label/input";
+    private String amazonPrimeLabel = "//*[@aria-label='Amazon Prime' or @class='a-size-base a-color-secondary']";
+
+    public ComputersAndAccessoriesPage(RemoteWebDriver driver){
+        super(driver);
+    }
+
+    public boolean isLoaded() {
+        return driver.findElement(By.xpath(pageTitle)).isDisplayed();
+    }
+
+    public String getTitle(){
+        driver.findElement(By.xpath(menuText));
+        return driver.findElement(By.xpath(pageTitle)).getText();
+    }
+
+    public ComputersAndAccessoriesPage setSearchCriteria(String searchText) {
+        getSearchField().sendKeys(searchText);
+        return this;
```

Comment:
is there any reason you are returning the class instance... seems to be redundant.